### ISSUE TITLE
Normalization in MLPicker

### ISSUE THREAD

**Issue by yetinam** [xdas-dev/xdas#36]
Hey!

I had a look at the normalization [code used by the MLPicker](https://github.com/xdas-dev/xdas/blob/71a5a9baebe9479b66609cc78529dc5e074daa6c/xdas/atoms/ml.py#L154). It seems that you always normalize with standard deviation. However, a good bunch of SeisBench models are trained with peak normalization. We had the same problem in SeisBench a while back (https://github.com/seisbench/seisbench/issues/187) and it turns out to be quite severe. This might also explain why DiTing for you works substantially better than the other models (it's trained on standard deviation).

An alternative would be to just use the `annotate_batch_pre` function of the model. This function automatically performs the normalization (in Pytorch) and uses the parameters from the config file that are loaded with `from_pretrained`.

**Comment by atrabattoni** [comment]
Good catch thank you! If `annotate_batch_pre` works on torch tensors I will definitely do that let me try. 

PS: we benchmarked the models with seisbench so it does not explain why DiTing performs better.

**Comment by atrabattoni** [comment]
Let see how seisbench work out on dealing with DAS data before doing anything else :)

**Comment by atrabattoni** [comment]
Closed by #66
